model_analyzer/triton/client/http_client.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+# Copyright (c) 2020-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
You can remove `(c)` to make the line shorter.

Reply:
I think it's fine to leave the `(c)` in to follow guidance from company, as long as this fits on one line with the auto-formatter, which it does.

Reply:
agreed

Reply:
The correct string is Copyright 20xx-20yy, NVIDIA CORPORATION & AFFILIATES. All rights reserved. So Tanmay is correct here.

Reply:
updated to remove `(c)` 👍